docs/learn/guides/messaging/messaging-with-sqs/README.md

```diff
@@ -0,0 +1,113 @@
+
+Here, the fire alarm is sending fire alerts periodically from a Ballerina worker where listener polls in another worker. Both sent messages and received messages are printed in the console. 
+
+As there can be multiple alert messages available in the queue, the listener is configured to consume more than one message at a time.
+
+The following diagram illustrates the scenario:
+
+![Message flow diagram image](resources/sqs-alert.png)
+
+
+## Prerequisites
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+  > **Tip**: For a better development experience, install one of the following Ballerina IDE plugins: [VSCode](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina), [IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+- [Amazon SQS Account](https://aws.amazon.com/sqs/)
+
+
+## Implementation
+> If you want to skip the basics and move directly to the [Testing](#testing) section, you can download the project from GitHub and skip the [Implementation](#implementation) instructions.
+
+Take a look at the code samples below to understand how to implement the scenario. 
+
+Following code creates a new queue in Amazon SQS with the configuration provided in a file.
+
+**create_notification_queue.bal**
+<!-- INCLUDE_CODE: guide/create_notification_queue.bal -->
+
+Following code generates fire alert notifications periodically and are sent to the above created SQS queue.
+
+**notify_fire.bal**
+<!-- INCLUDE_CODE: guide/notify_fire.bal -->
+
+Following code listens to the SQS queue and if there are any notifications, it would receive from the queue and delete the existing messages in the queue.
+
+**listen_to_fire_alarm.bal**
+<!-- INCLUDE_CODE: guide/listen_to_fire_alarm.bal -->
+
+In the following code, the main method would implement the workers related to creating a queue, sending a message to the queue consuming and receiving / deleting messages form the queue.
+
+**main.bal**
+<!-- INCLUDE_CODE: guide/main.bal -->
+
+### Creating the project structure
+
+Ballerina is a complete programming language that supports custom project structures. To implement the scenario in this guide, you can use the following package structure:
+
+```
+  messaging-with-sqs
+  |__guide
+        |
+        ├── create_notification_queue.bal
+        ├── notify_fire.bal
+        ├── listen_to_fire_alarm.bal
+        └── main.bal
+```
+ 
+- Create the above directories in your local machine and also create the empty .bal files.
+- Then open a terminal, navigate to the guide directory, and run the Ballerina project initializing toolkit.
+
+```bash
+   $ ballerina init
```

Comment:
This doesn't work anymore right?

Reply:
Yes as the Ballerina API has got changes all the guides seems broken.